pages/solidity-api/ChipRegistry.md

```diff
@@ -277,7 +278,7 @@ function addGatewayURL(string _gatewayUrl) external
 ```
 
 ONLY OWNER: Add a new gateway URL to the array of gateway URLs. This array returns different URLs the client can call to
-get the data to resolve an unclaimed chip. The client can then use the data returned from the URL to call resolveUnclaimedChip.
+get the data to resolve an unclaimed chip (TODO: what happens when it is claimed? what happens to these gateway URLs?). The client can then use the data returned from the URL to call resolveUnclaimedChip.
 
```

Comment:
Question here: 'what happens when it is claimed? what happens to these gateway URLs?)'